acceptance-tests/tests/workflows/accountManagementFlow.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { describe, beforeAll, it, expect, afterAll } from 'vitest';
+import { TestState } from '../../lib/TestState';
+import { getInitPromptSequence, ENTER } from '../../lib/prompt';
+
+describe('Account Management Flow', () => {
```

Comment:
This is an example of a workflow. It tests:
1. the config file shouldn't exist initially
2. It creates a config file and checks that it exists
3. It lists out the accounts and checks to see that the auth'd account is in the list
4. It lists out info for the account
5. It removes the account and makes sure that succeeded
6. It lists the accounts again to confirm that the account doesn't exist
7. It authenticates the account again and makes sure that it exists in the config